src/collections/_documentation/workflow/integrations/integration-platform/index.md

```diff
@@ -3,166 +3,569 @@ title: Integration Platform
+###  Installation
 
-### Resource Subscriptions
+Users will have the option to install your integrations on the Integrations Page in Sentry. If your integration is still in an unpublished state, the only Sentry organization that will be able to install that integration will be the organization that created the integration. Clicking 'Install' will allow users to see a description of your integration and the permissions that will be granted should the user choose to install. 
 
-Selecting a resource subscription determines the types of webhook requests sent to your service. Currently you can get requests for issues when they are created, assigned, ignored, or resolved. 
+[{% asset integration-platform-index/integrations.png alt="A list of integrations that a user can install." %}]({% asset integration-platform-index/integrations.png @path %})
 
-[{% asset integration-platform-index/resource-subscription.png alt="Screenshot of what the resource subscription looks like." %}]({% asset integration-platform-index/resource-subscription.png @path %})
+#### OAuth Process
 
-## Building the Integration
+After installation, if your user has approved of all permissions, Sentry will generate a grant code and an installation ID. This information, the grant code, and the installation ID are sent via the `installation.created` webhook to the Webhook URL specified in your configuration. 
 
-After installation, if your user has approved of all permissions, Sentry will generate a grant code and an installation ID. If your integration has a Redirect URL configured, the integration redirects the user's browser to the configured URL with the grant code and installation ID in the query params.
+However, if your integration has a Redirect URL configured, the integration redirects the user’s browser to the configured URL with the grant code and installation ID in the query params.
 
-Start your build by implementing the Redirect URL endpoint, `/setup` — typically where you exchange the grant code for an access token that allows you to make API calls to Sentry. 
+Start your build by implementing the Redirect URL endpoint, /setup — typically where you exchange the grant code for an access token that allows you to make API calls to Sentry.
 
 ```python
-import requests
-from flask import redirect, request
+    import requests
+    from flask import redirect, request
 
-@app.route('/setup', methods=['GET'])
-def setup():
-    code = request.args.get('code')
-    install_id = request.args.get('installationId')
-    
-    url = u'http://sentry.io/api/0/sentry-app-installations/{}/authorizations/'
+    @app.route('/setup', methods=['GET'])
+    def setup():
+        code = request.args.get('code')
+        install_id = request.args.get('installationId')
+        
+        url = u'https://sentry.io/api/0/sentry-app-installations/{}/authorizations/'
```

Comment:
@MeredithAnya @scefali What about the 'u' in front of this URL?

Reply:
@MimiDumpling Don't think we need it here either